llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -261,7 +261,9 @@ RISCVInstructionSelector::selectZExtBits(MachineOperand &Root,
     return {{[=](MachineInstrBuilder &MIB) { MIB.addReg(RegX); }}};
   }
 
-  // TODO: Use computeKnownBits.
+  unsigned Size = MRI->getType(RootReg).getSizeInBits();
```

Comment:
scalar size?

Reply:
It should always be a scalar I think. Unless this gets called before any of the type checks in the pattern. But I made the change.

Reply:
I guess it would assert in practice in the APInt constructor